modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/util/MetadataTypeUtils.java

```diff
@@ -133,4 +136,8 @@ public static boolean isExtensible(MetadataType metadataType) {
   public static boolean isEnum(MetadataType metadataType) {
     return metadataType.getAnnotation(EnumAnnotation.class).isPresent();
   }
+
+  public static Set<MetadataKey> getKeysFromContainer(MetadataKeysContainer metadataKeysContainer) {
+    return metadataKeysContainer.getKeys(metadataKeysContainer.getResolvers().iterator().next()).get();
```

Comment:
is no clear from the signature what this method does. I'd spectrum it to squash the keys from all the resolvrrs into a single set, not to take only the first. if this is for testing only then can add it to the basetestcase as util method